.ci/actions/os-preparation/action.yml

```diff
@@ -0,0 +1,25 @@
+name: 'OS Preparation'
+description: 'Prepares the Operating Systems array'
+
+outputs:
+  operating-systems:
+    description: Configured job operating systems
+    value: ${{ steps.set-os.outputs.operating_systems }}
+
+runs:
+  using: "composite"
+  steps:
+    - name: Check out droolsjbpm-build-bootstrap
+      uses: actions/checkout@v3
+      with:
+        repository: kiegroup/droolsjbpm-build-bootstrap
+        path: matrix_preparation
```

Comment:
I don't see where `matrix_preparation` is really use it, I guess the folder is missing on `set-os` step, right?

Reply:
Just thinking that it could be useless in this case, hence removing `path` field it should work in the same way

Reply:
yeah you are right, added to reference matrix.json as `$GITHUB_WORKSPACE/matrix_preparation/.ci/actions/os-preparation/matrix.json`